### PR TITLE
store: handle oopses, report id to user

### PR DESCRIPTION
**Pull request by chipaca**

### CODE REVIEW

**Comment by chipaca** [comment]
Also I might've renamed `store/snap_remote_repo.go → store/store.go`

File: store/store.go [hunk 1]
```diff
@@ -262,7 +262,12 @@ func (s *SnapUbuntuStoreRepository) Snap(name, channel string, auther Authentica
 	case resp.StatusCode == 404:
 		return nil, ErrSnapNotFound
 	case resp.StatusCode != 200:
-		return nil, fmt.Errorf("SnapUbuntuStoreRepository: unexpected HTTP status code %d while looking for snap %q/%q", resp.StatusCode, name, channel)
+		tpl := "Ubuntu CPI service returned unexpected HTTP status code %d while looking for snap %q/%q"
```
Comment (pedronis) [inline store/store.go]: given that we don't use the notation anymore on the command line  itself maybe time to change this to snap %q in channel %q ?

Comment (chipaca) [inline store/store.go]: good point.

**Comment by pedronis** [comment]
LGTM but remark about new and old error

**Comment by zyga** [comment]
Nice, LGTM

**Pull request merged**